### PR TITLE
Create 0720-okJiang-as-migration-committer.md

### PR DESCRIPTION
**Pull request by lance6716**

### CODE REVIEW

**Comment by lance6716** [comment]
/cc @amyangfei 

@zhangyangyu

**Comment by lance6716** [comment]
/hold

for 6 days

File: votes/0720-okJiang-as-migration-committer.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# A Vote for okJiang as Migration Committer
+
+## Proposal
+
+[okJiang](https://github.com/okJiang) has joined TiDB Data Migration team for more than 1 year. During this period, he contributed mainly to DM/Lightning/Dumpling, fixed bugs, and added sereral new features for them.
```
Comment (amyangfei) [inline votes/0720-okJiang-as-migration-committer.md:5]:
```suggestion
[okJiang](https://github.com/okJiang) has joined TiDB Data Migration team for more than 1 year. During this period, he contributed mainly to DM/Lightning/Dumpling, fixed bugs, and added several new features for them.
```

**Review by amyangfei — APPROVED** [review]

**Pull request merged**